test/blackbox-tests/test-cases/pkg/opam-package-with-extra-source.t

```diff
@@ -31,8 +31,15 @@ Make a package with an extra-source field
   > (source (copy $PWD/source))
   > EOF
 
-The lockfile should contain the fetching of extra sources. It currently does not. 
+The lockfile should contain the fetching of extra sources.
 
   $ cat dune.lock/with-extra-source.pkg 
   (version 0.0.1)
+  
+  (extra_sources
+   (some/file
+    (fetch
+     (url https://some-url)
+     (checksum
+      sha256=8beda92f97cde6d4a55a836ca6dc9f860bb5f1a6b765b80be4594943288571cf))))
```

Comment:
I forgot we haven't fixed the frontend for this yet. Definitely a little ugly.

Could you add an example with md5 as well?